### PR TITLE
Current epoch check remove

### PR DESCRIPTION
**Pull request by eesmerdag**

### CODE REVIEW

File: pipeline/task_fetcher.go [hunk 1]
```diff
@@ -226,16 +226,14 @@ func (t FetcherTask) Run(ctx context.Context, payload *Payload) error {
...
 
-				if data.FirstBlockOfNewEpoch {
-					logrus.WithField("height", data.Height).Info("fetching validator delegations")
-					delegations, err := t.fetchDelegations(data.Validators)
-					if err != nil {
-						return err
-					}
-					data.DelegationsByValidator = delegations
+				logrus.WithField("height", data.Height).Info("fetching validator delegations")
+				delegations, err := t.fetchDelegations(data.Validators, data.Height)
```
Comment (sosedoff) [inline pipeline/task_fetcher.go:231]: Does this mean we will be fetching delegation on all heights?

Comment (eesmerdag) [reply pipeline/task_fetcher.go:231]: no sir, calling this method is under the scope of processing at the first block of new epoch.

Comment (sosedoff) [reply pipeline/task_fetcher.go:231]: Lol i should have looked before asking

Comment (eesmerdag) [reply pipeline/task_fetcher.go:231]: 😬 yes sir. do you have any other concern?

**Pull request merged**